spec/rails_helper.rb

```diff
@@ -51,4 +59,25 @@
   config.filter_rails_from_backtrace!
   # arbitrary gems may also be filtered via:
   # config.filter_gems_from_backtrace("gem name")
+ # def login(email, password)
+ #   visit login_path
+ #   fill_in :email, with: email
+ #   fill_in :password, with: password
+ #   click_button "Login"
+ # end
+# module Sorcery
+# module TestHelpers
+ # module Rails
+  #    def login_user_post(user, password)
+   #     page.driver.post(user_sessions_url, { email: user.email, password: password}) 
+    #  end
+   # end
+#  end
+ # end
+  def login user, password = 'login'
```

Comment:
Use def with parentheses when there are parameters.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.